lib/matplotlib/mlab.py

```diff
@@ -1113,6 +1113,8 @@ def specgram(x, NFFT=None, Fs=None, detrend=None, window=None,
         noverlap = 128  # default in _spectral_helper() is noverlap = 0
     if NFFT is None:
         NFFT = 256  # same default as in _spectral_helper()
+    if Fs is None:
```

Comment:
`Fs` is only passed through in this function. It's not used explicitly. Therefore, there is no need to set the default explicitly again here; and I thus prefer to not duplicate the default. It's different with `NFFT` because its value is used in the warning.